Assets/Scripts/Towers/Tower.cs

```diff
@@ -0,0 +1,152 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Tower : MonoBehaviour {
+    public enum TargetingMethod {
+        HighestPriority,
```

Comment:
At first glance, I'm unsure what the "Highest" or "Lowest" priorities are. 
I wouldn't word it high/low, rather first in lane/last in lane

Reply:
That's not what those are, they're priorities given to unit types